CHANGELOG.rst

```diff
@@ -15,6 +21,30 @@ Changes
     was added to move to the `Service` model. It is advised to verify the `Service`
     instances in the admin to check that the data migration was ran as expected.
 
+**New features**
+
+* [#108] Admin action to check Abonnement callback status
+* [#180] Provide an admin overview for notificatie responses
+* [#207] Add experimental PUT and PATCH for Kanaal
+
+**Project maintenance**
+
+* [maykinmedia/objects-api#463] Add trivy image scan
+* [#199] Add Admin OIDC Configuration step from django-setup-configuration
+* [#204] Add SitesConfiguration Step from django-setup-configuration
+* [#200] Autorisaties-API configuration via django-setup-configuration
+* [#202] Configuration Kanalen via django-setup-configuration
+* [#202] Configuration Abonnementen via django-setup-configuration
+* [#203] Configuration Notification settings via django-setup-configuration
+* [maykinmedia/open-api-framework#92] Fix docker latest tag publish
+* [#207] Update register_kanalen to update existing kanalen
```

Comment:
This was changed in `notifications-api-common` and should probably be in the changelog of Open Zaak instead
```suggestion
```